### PR TITLE
Fixed issue 24

### PR DESCRIPTION
**Pull request by bmenne-dspace**
Fixed issue "Layered Standard Manifest File is missing for fmi-ls-bus"

### CODE REVIEW

File: docs/examples/fmi_ls_bus_manifest_example.xml [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<fmiLayeredStandardManifest
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:noNamespaceSchemaLocation="../../schema/fmi3LayeredStandardBusManifest.xsd"
+	xmlns:fmi-ls="http://fmi-standard.org/fmi-ls-manifest"
+	fmi-ls:fmi-ls-name="org.fmi-standard.fmi-ls-bus"
+	fmi-ls:fmi-ls-version="1.0.0"
+	fmi-ls:fmi-ls-description="Layered Standard for Bus"/>
```
Comment (PTaeuberDS) [inline docs/examples/fmi_ls_bus_manifest_example.xml:8]: I would prefer "Layered Standard for Network Communication" as the official name of the LS...
However, this is a description field and is probably displayed in the importing tool. Therefore, it should provide a little more information about the contents of this layered standard. I will create an issue for that here and also in the fmi-ls-xcp repo.

**Review by PTaeuberDS — APPROVED** [review]
Looks good to me

**Review by andreas-junghanns — CHANGES_REQUESTED** [review]
I think we should state the location of the manifest file within the FMU here in this standard as well.

**Comment by bmenne-dspace** [comment]
> I think we should state the location of the manifest file within the FMU here in this standard as well.

I

> I think we should state the location of the manifest file within the FMU here in this standard as well.

@andreas-junghanns Yes you are right, I will fix it and push the changes

**Review by andreas-junghanns — APPROVED** [review]

**Pull request merged**